web-common/src/features/entity-management/name-utils.ts

```diff
@@ -1,3 +1,8 @@
+export const nameRegex = /^[^<>:"/\\|?*]+$/;
```

Comment:
Global constants like this should probably have 1st char capitalised.

Reply:
Good point – though changed to ALL_CAPS